content/concepts/did-ddo.md

```diff
@@ -1,173 +1,699 @@
+  }
+}
+```
+
+### Datatokens
+
+The `datatokens` array contains information about the ERC20 datatokens attached to [asset services](#services).
+
+| Attribute       | Type     | Description                                      |
+| --------------- | -------- | ------------------------------------------------ |
+| **`address`**   | `string` | Contract address of the deployed ERC20 contract. |
+| **`name`**      | `string` | Name of NFT set in contract.                     |
+| **`symbol`**    | `string` | Symbol of NFT set in contract.                   |
+| **`serviceId`** | `string` | ID of the service the datatoken is attached to.  |
+
+Example:
 
 ```json
 {
-  "type": "credential3Box",
-  "values": ["profile1", "profile2"]
+  "datatokens": [
+    {
+      "address": "0x000000",
+      "name": "Datatoken 1",
+      "symbol": "DT-1",
+      "serviceId": "1"
+    },
+    {
+      "address": "0x000001",
+      "name": "Datatoken 2",
+      "symbol": "DT-2",
+      "serviceId": "2"
+    }
+  ]
+}
+```
+
+### Event
+
+The `event` section contains information about the last transaction that created or updated the DDO.
+
+Example:
+
+```json
+{
+  "event": {
+    "tx": "0x8d127de58509be5dfac600792ad24cc9164921571d168bff2f123c7f1cb4b11c",
+    "block": 12831214,
+    "from": "0xAcca11dbeD4F863Bb3bC2336D3CE5BAC52aa1f83",
+    "contract": "0x1a4b70d8c9DcA47cD6D0Fb3c52BB8634CA1C0Fdf",
+    "datetime": "2000-10-31T01:30:00"
+  }
+}
+```
+
+### Purgatory
+
+Contains information about an asset's purgatory status defined in [`list-purgatory`](https://github.com/oceanprotocol/list-purgatory). Marketplace interfaces are encouraged to prevent certain user actions like adding liquidity on assets in purgatory.
+
+| Attribute    | Type      | Description                                                                                   |
+| ------------ | --------- | --------------------------------------------------------------------------------------------- |
+| **`state`**  | `boolean` | If `true`, asset is in purgatory.                                                             |
+| **`reason`** | `string`  | If asset is in purgatory, contains the reason for being there as defined in `list-purgatory`. |
+
+Example:
+
+```json
+{
+  "purgatory": {
+    "state": true,
+    "reason": "Copyright violation"
+  }
+}
+```
+
+```json
+{
+  "purgatory": {
+    "state": false
+  }
+}
+```
+
+### Statistics
+
+The `stats` section contains different statistics fields.
+
+| Attribute      | Type     | Description                                                                                                   |
+| -------------- | -------- | ------------------------------------------------------------------------------------------------------------- |
+| **`consumes`** | `number` | How often an asset was consumed, meaning how often it was either downloaded or used as part of a compute job. |
+
+Example:
+
+```json
+{
+  "stats": {
+    "consumes": 4
```

Comment:
Please rename this in `orders` . We don't use  `consumes` anywhere and it doesn't reflect the reality. This value is the number of orders made on that asset.